.github/workflows/CI.yml

```
@@ -0,0 +1,74 @@
+name: CI
+# Run on master, any tag or any pull request
+on:
+  push:
+    branches:
+      - master
+      - staging
+      - trying
+    tags: '*'
+  schedule: 
+    - cron: '0 2 * * *'  # Daily at 2 a.m. UTC
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - 1
+        os:
+          - ubuntu-latest
+          - macOS-latest
+        arch:
+          - x64
+        include:
+          # Add a 1.0 job just to make sure we still support it
+          - os: ubuntu-latest
+            version: 1.0.5
+            arch: x64
```

If we put 1.0 in the `version` block then just delete this block, else we can change it to the following

```suggestion
          # Add a 1.0 job just to make sure we still support it
          - os: ubuntu-latest
            version: "1.0"
            arch: x64
```